src/database/sqlite/initialization.md

```diff
@@ -0,0 +1,42 @@
+## Create a SQLite database
+
+[![rusqlite-badge]][rusqlite] [![cat-database-badge]][cat-database]
+
+You can use the `rusqlite` crate to open SQLite databases. Note that, if you
```

Comment:
Personal pronouns should not be used in the description.  Refer to the code not the author or the audience.  In this case, just drop 'You can'